### ISSUE TITLE
Fail to reproduce the result of fig 1e in the paper

### ISSUE THREAD

**Issue by protein2019** [Illumina/SpliceAI#84]
Hi,

I use the models in spliceai/model and test code in https://basespace.illumina.com/s/5u6ThOblecrh, and find that the top-k accuracy is 0.9172, and PR-AUC is 0.9619. These metrics are lower than the result in fig 1e of the original paper, which is 0.95 for top-k accuracy and 0.98 for PR-AUC. Thus, I am wondering if I make some mistakes, or there are some differences between models at GitHub and models used in the paper. 

The whole output of test_model.py is as follows.
Acceptor:
0.9922	0.9131	0.9802	0.9880	0.9587	0.9862	0.6459	0.1070	0.0206	14289

Donor:
0.9938	0.9213	0.9846	0.9903	0.9650	0.9885	0.6554	0.1034	0.0184	14289

Thanks for your help in advance!

**Comment by kishorejaganathan** [comment]
Did you use the files inside the canonical folder or the gtex folder? For Fig 1, we used the files from the canonical folder.

**Comment by protein2019** [comment]
I used the files inside the canonical folder.

**Comment by kishorejaganathan** [comment]
Hmm that is surprising, many people have reproduced the results. Could you try adding a batch normalization layer just before the last layer and see if that helps? If that doesn't help, could you try downgrading the version of tensorflow/keras to 1.9/2.0.5?

**Comment by protein2019** [comment]
Thanks for your reply. Maybe I make some mistakes but I cannot find them. I use the models in GitHub spliceai/model directly, so there seems to be no way to add a layer. I try tensorflow 1.4.1 and keras 2.0.5, and find that the results are the same. 
I do the same as README.txt, by updating the constants.py file, and then run the following commands:
```
bash grab_sequence.sh

python create_datafile.py train all
python create_datafile.py test 0

python create_dataset.py train all
python create_dataset.py test 0

bash script_test.sh 10000
```
Maybe I should train the model again to reproduce the results?

**Comment by protein2019** [comment]
Oh, maybe I misunderstand the models in https://github.com/Illumina/SpliceAI/tree/master/spliceai/models? Are those models the results of script_train.sh in the Canonical folder, or are models generated in other ways? I use the models there directly to test.

**Comment by kishorejaganathan** [comment]
That explains it, those models are from the GTEx folder - they consistently perform well for variant interpretation but drop a couple of percentage points in the test set. If you want to reproduce the Fig 1e results, please use the scripts in the Canonical folder.

**Comment by protein2019** [comment]
Okay, get it. Thanks a lot.

**Comment by protein2019** [comment]
Hi, could you please give me a copy of the models from the Canonical folder? Since the random seed is not set in the code, it will be easier to reproduce the result if I can have these models directly.

**Comment by kishorejaganathan** [comment]
You should find them here: https://drive.google.com/file/d/1DrnOVmyLV2rFWWTa-lbZWzP3YzwO59K2/view?usp=sharing (c is canonical and g is gtex), let me know if you have trouble accessing them.

**Comment by protein2019** [comment]
Thanks! I have downloaded them and reproduced the results successfully.
